dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESIndexUtil.java

```diff
@@ -0,0 +1,81 @@
+package com.dotcms.content.elasticsearch.business;
+
+import com.dotcms.content.elasticsearch.util.RestHighLevelClientProvider;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.util.Logger;
+import com.rainerhahnekamp.sneakythrow.Sneaky;
+import org.elasticsearch.action.admin.indices.settings.get.GetSettingsRequest;
+import org.elasticsearch.action.admin.indices.settings.get.GetSettingsResponse;
+import org.elasticsearch.action.admin.indices.settings.put.UpdateSettingsRequest;
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.common.settings.Settings;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class ESIndexUtil {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)